### PR TITLE
Refactor susy-grid to css-grid

### PR DESCRIPTION
**Pull request by dreampulse**
# Definition of Done

- [x] The code does at least what is defined in the ticket and does not affect any other functionality
- [x] The PR has a meaningful title
- [x] There are no 'hacks' or shortcuts refactoring is preferred
- [x] The happy case of the app was tested at least once manually

### CODE REVIEW

File: src/app/component/container/_container.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
   position: relative;
   width: 100%;
   margin: 0 auto;
-  padding: 0 gutter();
-  max-width: container();
+  padding: 0 $gutter-width / 2;
```
Comment (webholics) [inline src/app/component/container/_container.scss]: this is a bug, we should talk

File: src/sass/_config.scss [hunk 2]
```diff
@@ -83,22 +83,9 @@ $space-xxxxl: 100px;
 $global-max-width: 1170px;
 
 $columns: 12;
```
Comment (webholics) [inline src/sass/_config.scss]: this is never used

File: src/sass/_config.scss [hunk 3]
```diff
@@ -83,22 +83,9 @@ $space-xxxxl: 100px;
...
-$gutter-width: 30px;
+$gutter-width: $space-xl;
 $column-width: $global-max-width / $columns - $gutter-width;
```
Comment (webholics) [inline src/sass/_config.scss]: this is never used

File: src/app/component/container/_container.scss (hunk 1 above)
Comment (dreampulse) [reply src/app/component/container/_container.scss]: `$space-base`

**Pull request merged**